x-pack/plugins/fleet/public/applications/fleet/layouts/error.tsx

```diff
@@ -71,7 +71,7 @@ export const PermissionsError: React.FunctionComponent<{
               ) : (
                 <FormattedMessage
                   id="xpack.fleet.permissionDeniedErrorMessage"
-                  defaultMessage="You are not authorized to access Fleet. It requires the {roleName1} Kibana privilege for Fleet, and the {roleName2} or {roleName1} privilege for Integrations."
+                  defaultMessage="You are not authorized to access Fleet. It requires the {roleName1} Kibana privilege for Fleet and its Sub-features, and the {roleName2} or {roleName1} privilege for Integrations."
```

Comment:
is this accurate? I think only _some_ sub-feature privileges are needed to access fleet (not `all`). maybe we can word it like this:

```suggestion
                  defaultMessage="You are not authorized to access Fleet. It requires Kibana privileges for Fleet, and the {roleName2} or {roleName1} privilege for Integrations."
```

@kilfoyle maybe you can wordsmith this better 😅

Reply:
I can try! How about `You are not authorized to access Fleet. Kibana privileges are required to access Fleet; the {roleName2} or {roleName1} privilege is required to access Integrations.`